variables.tf

```diff
@@ -389,6 +389,12 @@ variable "gke_monitoring_service" {
   default     = "none"
 }
 
+variable "enable_managed_prometheus" {
+  type        = bool
+  description = "Enable Google Cloud Managed Service for Prometheus for your cluster"
+  default     = "false"
```

Comment:
false shouldn't be quoted as it's a Boolean value.

Reply:
fixed in latest push